packages/frontend/src/features/catalog/components/CatalogMetadata.tsx

```diff
@@ -428,12 +436,60 @@ export const CatalogMetadata: FC = () => {
                             },
                         })}
                         onClick={() => {
-                            history.push(
+                            if (metadata && isViewingField) {
+                                return history.push(
+                                    getExplorerUrlFromCreateSavedChartVersion(
```

Comment:
This whole block is basically just a hardcoded default chart right? I feel like this should be a constant somewhere instead of inline in this handler. Maybe a util like getDefaultExplore? OR getExplorerUrlFromCreateSavedChartVersion could take null as a second argument and if so, it uses this default chart?

This is kinda hygiene so it doesn't have to be a blocker, but someday someone is gonna be confused about where this explore is getting initialized 😄

Reply:
100% @magnew ! 

and here it is: https://github.com/lightdash/lightdash/pull/10509/commits/869b2bd7453946bc6b69e4308eca7d87dd28e33c